chore: Add mention of support ticket when opening a pull request (#2507)
mongodb/terraform-provider-mongodbatlas: contributing/development-setup.md

- Make sure that the PR title follows [*Conventional Commits*](https://www.conventionalcommits.org/).
- Add comments around your new code that explain what's happening.
- Commit and push your changes to your branch then submit a pull request against the `master` branch.

> - A repo maintainer will review your pull request. **Note**: If you have an active MongoDB Atlas Support contract, you can optionally create a support ticket to address questions on this process.

Reviewer: ```suggestion
- A repo maintainer will review your pull request. **Note**: If you have an active [MongoDB Atlas Support ](https://www.mongodb.com/services/support/atlas-support-plans)contract, we recommend also creating a support ticket for any questions related to this process. .
```
